src/Inputs.ts

```diff
@@ -129,6 +130,14 @@ export class CoreInputs implements Inputs {
         return preRelease == 'true'
     }
 
+    private static get omitPrereleaseDuringUpdate(): boolean {
+        return core.getInput('omitPrereleaseDuringUpdate') == 'true'
+    }
+
+    get updatedPrerelease(): boolean | undefined {
+        if (CoreInputs.omitPrereleaseDuringUpdate) return undefined
+        return this.prerelease
+    }
```

Comment:
Could we also update the `prerelease` getter to `createdPrerelease` so it's consistent with the other inputs that have `created` vs `updated` variations?